cypress/e2e/settings/tenant/verify-item-status-language-when-locale-is-german.cy.js

```diff
@@ -0,0 +1,81 @@
+import Helper from '../../../support/fragments/finance/financeHelper';
+import { Permissions } from '../../../support/dictionary';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import TenantPane, { TENANTS } from '../../../support/fragments/settings/tenant/tenantPane';
+import Users from '../../../support/fragments/users/users';
+import { randomFourDigitNumber } from '../../../support/utils/stringTools';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import { Localization } from '../../../support/fragments/settings/tenant/general';
+import TemporarySessionLocale from '../../../support/fragments/settings/developer/session-locale/temporarySessionLocale';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+
+describe('Settings: Tenant', () => {
```

Comment:
Please use discribes like describe('inventory', () => {
  describe('Item', () => {

Reply:
Done in a way of: describe('Settings', () => {
describe('Tenant', () => {